mmengine/analysis/print_helper.py

```diff
@@ -0,0 +1,726 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+# Modified from
+# https://github.com/facebookresearch/fvcore/blob/main/fvcore/nn/print_model_statistics.py
+
+from collections import defaultdict
+from typing import Any, Dict, Iterable, List, Optional, Set, Tuple
+
+import tabulate
+import torch
+from torch import nn
+
+from .complexity_analysis import (ActivationAnalyzer, FlopAnalyzer,
+                                  parameter_count)
+
+
+def _format_size(x: int, sig_figs: int = 3, hide_zero: bool = False) -> str:
+    """Formats an integer for printing in a table or model representation.
+
+    Expresses the number in terms of 'kilo', 'mega', etc., using
+    'K', 'M', etc. as a suffix.
+
+    Args:
+        x (int) : The integer to format.
+        sig_figs (int) : The number of significant figures to keep
+        hide_zero (bool) : If True, x=0 is replaced with an empty string
+            instead of '0'.
+
+    Returns:
+        str : The formatted string.
```

Comment:
```suggestion
    Args:
        x (int): The integer to format.
        sig_figs (int): The number of significant figures to keep.
            Defaults to 3.
        hide_zero (bool): If True, x=0 is replaced with an empty string
            instead of '0'. Defaults to False.

    Returns:
        str: The formatted string.
```